hudi-client/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -177,44 +175,7 @@ protected void finalizeWrite(HoodieTable<T> table, String instantTime, List<Hood
 
   private void updateMetadataAndRollingStats(String actionType, HoodieCommitMetadata metadata,
```

Comment:
Since, this method is just callling writeStats.forEach(stat -> metadata.addWriteStat(stat.getPartitionPath(), stat)); , can we remove this function altogether and move "writeStats.forEach(stat -> metadata.addWriteStat(stat.getPartitionPath(), stat));" to the caller.

Reply:
Good suggestion, fixed